lib/graph3d/Graph3d.js

```diff
@@ -327,7 +327,25 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
       distinctValues.push(data[i][column]);
     }
   }
-  return distinctValues;
+  return distinctValues.sort(function(a,b) { return a - b; });
+}
+
+
+Graph3d.prototype.getSmallestDifference = function(data, column) {
```

Comment:
Open for discussion: 

Is space complexity a matter here? I assume `data` isn't usually that large right? Otherwise, I might prefer something like:

`Sort data`
`Calc minDiff <- data[1] - data[0]`
`for (i <- 2 to len(a)) do minDiff <- min(minDiff, data[i] - data[i-i])`

Also maybe mention in a comment that when you pass an empty `data` value the function returns `undefined`.